server/server/src/config.rs

```diff
@@ -151,13 +129,52 @@ impl MetricsConfig {
     }
 }
 
+#[derive(Clone)]
+pub struct BillingConfig {
```

Comment:
I don't mean organize via prefixes via variable names, I mean anything common stays in this struct, anything related to stripe can get organized into it's own struct, etc.

Reply:
Isn't that what I had before? :joy: